package/libnetconf2/libnetconf2.mk

```diff
@@ -9,15 +9,15 @@ LIBNETCONF2_SITE = $(call github,CESNET,libnetconf2,v$(LIBNETCONF2_VERSION))
 LIBNETCONF2_INSTALL_STAGING = YES
 LIBNETCONF2_LICENSE = BSD-3-Clause
 LIBNETCONF2_LICENSE_FILES = LICENSE
-LIBNETCONF2_DEPENDENCIES = libyang libcurl
-HOST_LIBNETCONF2_DEPENDENCIES = host-libyang host-libcurl
+LIBNETCONF2_DEPENDENCIES = libyang libopenssl libcurl
```

Comment:
Missing libssh?

Reply:
Yup, thanks!!  This is why I looove reviews

Reply:
Turns out that dep was added further down in the .mk file already.  But I found another thing in the Config.in: seems we should select `BR2_PACKAGE_LIBOPENSSL` and not `BR2_PACKAGE_OPENSSL`, the latter seems to be the meta package we talked about earlier.

Fixing that up now.